flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlCreateTableConverter.java

```diff
@@ -171,11 +171,7 @@ private CatalogTable createCatalogTable(SqlCreateTable sqlCreateTable) {
                         mergingStrategies);
         verifyPartitioningColumnsExist(mergedSchema, partitionKeys);
 
-        String tableComment =
-                sqlCreateTable
-                        .getComment()
-                        .map(comment -> comment.getValueAs(NlsString.class).getValue())
-                        .orElse(null);
+        String tableComment = OperationConverterUtils.getTableComment(sqlCreateTable.getComment());
```

Comment:
I'm wondering it works when the comment is Chinese? Can you please verify with Chinese comment?
I mean, will it cause the problem like 
https://issues.apache.org/jira/browse/FLINK-32249?

Reply:
Thank you for the reminder.
I tested it in both UT and sql-client and it passed.
![image](https://github.com/apache/flink/assets/9247964/4237f611-5f96-42f7-bf2f-fc7dddc21566)

![image](https://github.com/apache/flink/assets/9247964/173a4088-38fd-465f-9cd6-e3fc54ff4abc)